ppinot-model/src/main/java/es/us/isa/ppinot/evaluation/computers/TimeMeasureComputer.java

```diff
@@ -292,12 +292,19 @@ protected double computeValue() {
         public void ends(DateTime end) {
             if (isRunning()) {
                 duration = new DurationWithExclusion(start, end, ((TimeMeasure)definition).getConsiderOnly());
+                if (((TimeMeasure)definition).isFirstTo()) {
+                    this.reset();
```

Comment:
I wouldn't use reset for this because if you do so, you can have problems if there is a second start (start would be null again and a new duration will be computed).

Instead, I think you should add a new attribute in the class (e.g. boolean stopped) as a flag so that new ends are ignored.

Reply:
That is true, I did not realize that. I will make the change.